docs/source/rescue.md

```diff
@@ -0,0 +1,56 @@
+| variant_count         | The threshold for the minimal read depth of a variant in the cell group when subseting the informative variants by default. Default: 10                                                                                                                                 |
+| variant_pct           | The threshold for the minimal frequency of the alternative or reference allele to determine the dominant allele of a variant in the cell group when subseting the informative variants by default. Default: 0.9                                                         |
+| vireo_parent_dir      | A parent folder which contains the output folder of vireo in the format of `vireo_[taskID/sampleId]` generated by hadge pipeline only when running in donor_match mode. In other modes, the input is passed by the pipeline automatically. Default: None                |
+
+## **Output**
+
+By default, the pipeline is run on a single sample. In this case, all pipeline output will be saved in the folder `$projectDir/$params.outdir/rescue`. When running the pipeline on multiple samples, the pipeline output will be found in the folder `"$projectDir/$params.outdir/$sampleId/rescue`. To simplify this, we'll refer to this folder as `$pipeline_output_folder` from now on.
+
+In rescue mode, the genotype- and hashing-based demultiplexing workflow run in parallel. They save their output in `$pipeline_output_folder/[gene/hash]_demulti`. Before running the donor-matching preocess, the pipeline merges the results of two workflows into `classification_all_genetic_and_hash.csv` and `assignment_all_genetic_and_hash.csv` in the `$pipeline_output_folder/summary` folder.
+
+The following additional output can be found in `$pipeline_output_folder/donor_match`.
+
+### Optional output: Donor matching
+
+- Folder`[method1]_[task_ID/sampleId]_vs_[method2]_[task_ID/sampleId]` with:
+  - `correlation_res.csv`: correlation scores of donor matching
+  - `concordance_heatmap.png`: a heatmap visualising the the correlation scores
+  - `donor_match.csv`: a map between hashtag and donor identity.
+  - `all_assignment_after_match.csv`: assignment of all cell barcodes after donor matching
+  - `intersect_assignment_after_match.csv`: assignment of joint singlets after donor matching
+- General output in the `$pipeline_output_folder/donor_match` folder:
+  - `all_assignment_after_match.csv`: assignment of all cell barcodes based on the donor matching of the optimal match
+  - `donor_match.csv`: a map between hashtags and donor identities based on the donor matching of the optimal match
+  - `score_record.csv`: a CSV file storing the matching score and the number of matched donors for each method pair
+
+### Optinal output: scverse compatibility
+
+Folder `data_output` with:
```

Comment:
didn't you say above that the folder is called `adata`?

Reply:
so here I only mentioned the additional adata output in the rescue mode compared with the genetic/hashing mode